ui/pages/settings/networks-tab/networks-form/networks-form.test.js

```diff
@@ -1,10 +1,42 @@
 import React from 'react';
 import configureMockStore from 'redux-mock-store';
-import { fireEvent } from '@testing-library/react';
+import { fireEvent, screen } from '@testing-library/react';
+import nock from 'nock';
 import { renderWithProvider } from '../../../../../test/jest/rendering';
 import { defaultNetworksData } from '../networks-tab.constants';
+import { MAINNET_RPC_URL } from '../../../../../shared/constants/network';
 import NetworksForm from '.';
 
+nock('https://chainid.network:443', { encodedQueryParams: true })
```

Comment:
This will create a global mock across all tests. I'm afraid this will lead to intermittently failing tests. Should this go within a `beforeEach`, and does it makes sense to call `nock.cleanAll()` within an `afterEach`?

Reply:
Yeah good call out!

Reply:
done here @mcmire: https://github.com/MetaMask/metamask-extension/pull/12431/commits/8944afe5c5b3057550529316cd420b33775e61da